### PR TITLE
Fixed: Cleanse AuthKey followed by Double Quote

### PR DESCRIPTION
**Pull request by bakerboy448**
#### Database Migration
NO

#### Description
&authkey=xxxxx"
#### Todos
- [X] Tests
-  Translation Keys
-  Wiki Updates

#### Issues Fixed or Closed by this PR

* Fixes #XXXX

### CODE REVIEW

File: src/NzbDrone.Common.Test/InstrumentationTests/CleanseLogMessageFixture.cs [hunk 1]
```diff
@@ -19,6 +19,7 @@ public class CleanseLogMessageFixture
         [TestCase(@"http://nzb.su/getnzb/2b51db35e1912ffc138825a12b9933d2.nzb&i=37292&r=2b51db35e1910123321025a12b9933d2")]
         [TestCase(@"https://horrorcharnel.org/takeloginhorror.php: username=mySecret&password=mySecret&use_sslvalue==&perm_ssl=1&submitme=X&use_ssl=1&returnto=%2F&captchaSelection=1230456")]
         [TestCase(@"https://torrentdb.net/login: _token=2b51db35e1912ffc138825a12b9933d2&username=mySecret&password=mySecret&remember=on")]
+        [TestCase(@"feeds.php?feed=torrents_lossless24&amp;user=mySecret&amp;auth=mySecret&amp;passkey=mySecret&amp;authkey=2b51db35e1910123321025a12b9933d2")]
```
Comment (bakerboy448) [inline src/NzbDrone.Common.Test/InstrumentationTests/CleanseLogMessageFixture.cs:22]:
```suggestion
        [TestCase(@"feeds.php?feed=torrents_lossless24&amp;user=mySecret&amp;auth=mySecret&amp;passkey=mySecret&amp;authkey=2b51db35e1910123321025a12b9933d2""")]
```

**Review by bakerboy448 — COMMENTED** [review]
fix quotes

File: src/NzbDrone.Common.Test/InstrumentationTests/CleanseLogMessageFixture.cs [hunk 2]
```diff
@@ -19,6 +19,7 @@ public class CleanseLogMessageFixture
         [TestCase(@"http://nzb.su/getnzb/2b51db35e1912ffc138825a12b9933d2.nzb&i=37292&r=2b51db35e1910123321025a12b9933d2")]
         [TestCase(@"https://horrorcharnel.org/takeloginhorror.php: username=mySecret&password=mySecret&use_sslvalue==&perm_ssl=1&submitme=X&use_ssl=1&returnto=%2F&captchaSelection=1230456")]
         [TestCase(@"https://torrentdb.net/login: _token=2b51db35e1912ffc138825a12b9933d2&username=mySecret&password=mySecret&remember=on")]
+        [TestCase(@"feeds.php?feed=torrents_lossless24&amp;user=mySecret&amp;auth=mySecret&amp;passkey=mySecret&amp;authkey=2b51db35e1910123321025a12b9933d2""")]
```
Comment (Qstick) [inline src/NzbDrone.Common.Test/InstrumentationTests/CleanseLogMessageFixture.cs:22]: This test case passes without the change below... what are we missing?

**Comment by bakerboy448** [comment]
weird seems you're right

swear this was seen in the wild?

**Comment by Qstick** [comment]
> weird seems you're right
> 
> swear this was seen in the wild?

Yea thought so as well, if you can find it again we can fix it up.

**Pull request closed without merging**